### ISSUE TITLE
(placeholder) 404 error from Gravatar

### ISSUE THREAD

**Issue by kieckhafer** [reactioncommerce/reaction#3707]
PLACEHOLDER UNTIL I CAN REPLICATE AGAIN

### Actual behavior
Gravatar throws a 404 when.... ??

![grav](https://user-images.githubusercontent.com/4482263/36013692-9e46762a-0d1a-11e8-889d-8539a5e06fda.jpg)

### Versions

```
Node: 9.5.0
NPM: 5.6.0
Meteor Node: 8.9.3
Meteor NPM: 5.5.1
Reaction CLI: 0.27.0
Reaction: 1.8.0
Reaction branch: release-1.8.0
Docker: 18.01.0-ce
```

### QA

QA by @kieckhafer + @cassytaylor

**Comment by spencern** [comment]
We need reproduction steps for this issue.

**Comment by spencern** [comment]
This seems to be caused when using an email address which is not associated with a gravatar account

**Comment by willmoss1000** [comment]
Could I ask if this has been resolved?

**Comment by zenweasel** [comment]
@willmoss1000 I don't think this is resolved, but I have never seen it cause anything but a 404 in the dev console. Are you sure that's the same problem?

**Comment by willmoss1000** [comment]
@zenweasel yes pretty sure - I saw a 404 for secure.gravatar.com in the dev console. The notifications button disappeared, and the sign-in box showed as logged out when already logged in.

However I can't reproduce this for now, and it's not a major issue.  When it comes up again I'll post a screenshot, and note the circumstances.
